.docs/README.md

```diff
@@ -102,6 +114,17 @@ $mailer = new CompositeMailer([
 ]);
 ```
 
+### TraceableMailer
+
+It shows mail headers and mail preview (Edge doesn't support `iframe srcdoc` so it will show nothing) in tracy panel and delegates mail to provided mailer.
+
+TraceableMailer does only work with `showTracyPanel` option
+```yaml
+post:
+    showTracyPanel: %debugMode%
```

Comment:
`showTracyPanel` combination with `TraceableMailer` is little weird. I would prefer only one way. 

showTracyPanel (debug) -> internally wrap current mailer (preferable way)
or 
define mailer by yourself as you do in example